src/plugins/usage_collection/server/usage_counters/usage_counters_service.ts

```diff
@@ -66,7 +68,7 @@ export class UsageCountersService {
   constructor({ logger, retryCount, bufferDurationMs }: UsageCountersServiceDeps) {
     this.logger = logger;
     this.retryCount = retryCount;
-    this.bufferDurationMs = bufferDurationMs;
+    this.bufferDurationMs = 500;
```

Comment:
🚨 mind to revert this when you're done testing :)

Reply:
Oops good catch!